### PR TITLE
Add conda-sphinx-theme to conda-build 3.27.x

### PR DESCRIPTION
**Pull request by travishathaway**
### Description

Fixes: #5061 

This is the pull request for the 3.27.x branch.

### Checklist - did you ...

- [ ] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
- [ ] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

**Review by kathatherine — DISMISSED** [review]
LGTM as a first step

File: docs/source/conf.py [hunk 1]
```diff
@@ -104,23 +105,60 @@
...
 html_extra_path = ["robots.txt"]
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
+# html_static_path = ["_static"]
+
+# Custom CSS rules
+# html_style = "css/custom.css"
```
Comment (kenodegard) [inline docs/source/conf.py:161]: Delete `custom.css` too as is done in https://github.com/conda/conda-build/pull/5067?

File: docs/source/resources/compiler-tools.rst [hunk 2]
```diff
@@ -394,6 +394,71 @@ not available. You'd need to create a metapackage ``m2w64-gcc_win-64`` to
...
+expectations about the standard library in a unified way, and without
+implicitly depending on some global assumption about what the lower version
+on a given platform must be.
+
+In principle, this facility would make it possible to also express the
+dependence on separate stdlib implementations (like ``musl`` instead of
+``glibc``), or to remove the need to assume that a C++ compiler always needs to
+add a run-export on the C++ stdlib -- it could then be left up to packages
+themselves whether they need ``{{ stdlib('cxx') }}`` or not.
+
```
Comment (kenodegard) [inline docs/source/resources/compiler-tools.rst:461]: Not released yet (https://github.com/conda/conda-build/commit/e2916a2a5c590382323de97da6ac2fe6c516ff89):

```suggestion
```

File: docs/source/resources/define-metadata.rst [hunk 3]
```diff
@@ -1928,10 +1923,10 @@ variables are booleans.
    * - osx
      - True if the platform is macOS.
    * - arm64
-     - True if the platform is macOS and the Python architecture
-       is arm64.
+     - True if the platform is either macOS or Windows and the
+       Python architecture is arm64.
    * - unix
-     - True if the platform is either macOS or Linux.
+     - True if the platform is either macOS or Linux or emscripten.
```
Comment (kenodegard) [inline docs/source/resources/define-metadata.rst:1929]: Not released yet (https://github.com/conda/conda-build/commit/d480c32597d7cb986ec89dbc8152d474b627e404)

File: docs/source/resources/define-metadata.rst [hunk 4]
```diff
@@ -1965,6 +1960,11 @@ The use of the Python version selectors, `py27`, `py34`, etc. is discouraged in
 favor of the more general comparison operators.  Additional selectors in this
 series will not be added to conda-build.
 
+Note that for each subdir with OS and architecture that `conda` supports,
+two preprocessing selectors are created for the OS and the architecture separately
+except when the architecture is not a valid python expression (`*-32` and `*-64`
+in particular).
+
```
Comment (kenodegard) [inline docs/source/resources/define-metadata.rst:1967]: Not released yet (https://github.com/conda/conda-build/commit/d480c32597d7cb986ec89dbc8152d474b627e404)

**Review by kenodegard — CHANGES_REQUESTED** [review]

**Review by kenodegard — APPROVED** [review]

**Pull request merged**